### DISCUSSION TITLE
¿Como llenar el formato de entrega?

### DISCUSSION THREAD

**Discussion by Aragui** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#227] [Q&A] [answered]
Entiendo que las primeras 5 preguntas se refieren a cada una de las actividades, pero la ultima es el repo donde se alojan o es el repo de la practica de HTML?

**Comment by davidtoString** [comment, marked as answer]
Es el repo donde tienes todas las practicas ( requerimeintos, wireframes, publico objetivo)
Aun no dan el enlace del formualrio para las practicas de HTML.

**Comment by Aragui** [comment]
Gracias, al final meti mis archivos en el mismo repo :p
